eng/Npm.Workspace.nodeproj

```diff
@@ -72,6 +72,20 @@
     <Message Importance="High" Text="$(MSBuildProjectName) -> %0a%20%20$(_NpmFiles)" />
   </Target>
 
+  <Target Name="RestorePackageJsonFiles" AfterTargets="Build;Pack">
+    <Message Text="Restoring package.json files and package-lock.json from backups..." Importance="high" />
+    <ItemGroup>
+      <PackageJsonBackupFiles Include="$(RepoRoot)**/*.json.bak" />
+    </ItemGroup>
+    <Move SourceFiles="@(PackageJsonBackupFiles)"
+          DestinationFiles="@(PackageJsonBackupFiles -> Replace('.bak', ''))"
+          Condition="@(PackageJsonBackupFiles) != ''" />
+    <!-- Restore package-lock.json from git if it was modified -->
```

Comment:
instead of trying to restore these files, wouldn't it be simpler to call `git checkout -- <<files>>` using the exec task to undo the changes?